capplets/accessibility/at-properties/main.c

```diff
@@ -83,7 +92,26 @@ cb_mouse_preferences (GtkDialog *dialog, gint response_id)
 static void
 cb_login_preferences (GtkDialog *dialog, gint response_id)
 {
-	g_spawn_command_line_async ("mdmsetup", NULL);
+    DMType dm_type;
+    gchar *prog = NULL;
+
+    dm_type = dm_get_type();
+    if (dm_type == DM_TYPE_MDM) {
+        prog = g_find_program_in_path ("mdmsetup");
+    } else if (dm_type == DM_TYPE_LIGHTDM) {
+        prog = g_find_program_in_path ("lightdm-settings");
+        if (prog != NULL) {
+            g_free (prog);
+            prog = g_strconcat ("pkexec ", "lightdm-settings", NULL);
+        } else {
+            prog = g_find_program_in_path ("lightdm-gtk-greeter-settings-pkexec");
+        }
```

Comment:
as the tests become more and more complex, I think this should be factored out in a separate function and called here and from `create_builder()` instead of duplicating the logic.